client/scripts/controllers/app/recent_activity.ts

```diff
@@ -70,6 +68,54 @@ class RecentActivityController {
     return this._activeUsers;
   }
 
+  public async getRecentCommunityActivity(options: {
+    chainId: string;
+    communityId: string;
+    cutoffDate?: moment.Moment;
+  }): Promise<Array<OffchainThread[] | OffchainComment<any>[]>> {
+    const { chainId, communityId } = options;
+    const cutoffDate =
+      options.cutoffDate || moment(Date.now() - 30 * 24 * 3600 * 1000);
+
+    const params = {
+      chain: chainId,
+      community: communityId,
+      cutoff_date: cutoffDate.toISOString(),
+      include_threads: 't',
+    };
+    const response = await $.get(`${app.serverUrl()}/activeThreads`, params);
+    if (response.status !== 'Success') {
+      throw new Error(`Unsuccessful: ${response.status}`);
+    }
+    const { threads, comments } = response.result;
+
+    for (const comment of comments) {
```

Comment:
why are having the comments returned to the controller necessary for the summary page? Can't they just be rendered when we go into the proposal/thread itself? (I understand a count should be made, but that should be returned from the route as just an integer included with the thread)

Reply:
Our existing controller function to calculate threads' "last-updated" date uses comments, but I can just grab calculate that info on the backend + pass up a date/comment count instead of comments themselves.